rsmtool/comparer.py

```diff
@@ -123,32 +123,15 @@ def _modify_eval_columns_to_ensure_version_compatibilty(df,
         smd_name = 'SMD'
 
         # previously, the QWK metric used `trim_round` scores; now, we use just `trim` scores;
-        # to ensure backward compatibility, we check whether `trim_round` scores were used and
-        # if so, we update the rename dictionary and column lists accordingly
+        # We check whether `trim_round` scores were used and
+        # raise an error if this is the case
         if any(col.endswith('trim_round') for col in df if col.startswith('wtkappa')):
-
-            # replace the `wtkappa_*_trim` with `wtkappa_*_trim_round`
-            rename_dict_new = {(key.replace('_trim', '_trim_round')
-                                if key.startswith('wtkappa') else key):
-                               (val.replace("(b)", "(br)")
-                                if val.startswith('QWK') else val)
-                               for key, val in rename_dict_new.items()}
-
-            # replace the `wtkappa_*_trim` with `wtkappa_*_trim_round`
-            existing_eval_cols_new = [col.replace('_trim', '_trim_round')
-                                      if col.startswith('wtkappa') else col
-                                      for col in existing_eval_cols_new]
-
-            # replace the `QWK(b)` with `QWK(br)`
-            short_metrics_list_new = [col.replace("(b)", "(br)")
-                                      if col.startswith('QWK') else col
-                                      for col in short_metrics_list_new]
-
-            if raise_warnings:
-                warnings.warn("Please note that newer versions of RSMTool (7.0 or greater) use only the trimmed "
-                              "scores for weighted kappa calculations. Comparisons with experiments using "
-                              "`trim_round` for weighted kappa calculations will be deprecated in the next "
-                              "major release.", category=DeprecationWarning)
+            raise ValueError("Please note that newer versions of RSMTool (7.0 or greater) use only "
+                             " the trimmed scores for weighted kappa calculations. At least one "
+                             "of your experiments was created "
+                             "using an older version of RSMTool and uses trim_round for  "
+                             "`trim_round` for weighted kappa calculations. Please re-run "
+                             "the experiments with the recent version of RSMTool.")
```

Comment:
How about?

```suggestion
            raise ValueError("RSMTool (7.0 or greater) uses only "
                             "trimmed scores for weighted kappa calculations. At least one "
                             "of your experiments was run "
                             "using an older version of RSMTool that used  "
                             "trimmed and rounded scores instead. Please re-run "
                             "the experiment with the latest version of RSMTool.")
```

Reply:
I replaced "trimmed" with "unrounded" since this is actually more accurate. Let me know if it's not ok.